lib/api/controllers/authController.js

```diff
@@ -35,6 +35,9 @@ const { NativeController } = require('./baseController');
 const formatProcessing = require('../../core/auth/formatProcessing');
 const User = require('../../model/security/user');
 const ApiKey = require('../../model/storage/apiKey');
+const { generateRandomName } = require('../../util/name-generator');
+const { v4: uuidv4 } = require('uuid');
```

Comment:
This import should be next to the external packages imports l.27 and l.28.

The module imports should be separated in 3 parts:
```
// internal node modules like 'fs'

// external packages like 'lodash'

// local packages like './services/user'
```